test.el

```diff
@@ -275,11 +275,34 @@ that buffer once BODY finishes."
   (with-temp-buffer
     (insert-file-contents
      (expand-file-name "testdata/fill.BUILD" bazel-test--directory))
-    (bazel-mode)
-    (search-forward "# The Foobar files")
-    (let ((before (buffer-string)))
-      (fill-paragraph)
-      (should (equal (buffer-string) before)))))
+    (let ((original-buffer (current-buffer)))
+      (dolist (comment-text
+               '("keep sorted"
+                 "KEEP SORTEd"
```

Comment:
Do these comments generally ignore case? If so, the search should bind `case-fold-search` explicitly (probably in a different PR).

Reply:
They do, apparently.  See: https://github.com/bazelbuild/buildtools/blob/4.0.1/build/rewrite.go#L109-L118

I set `case-fold-search` to `t` in `bazel--find-magic-comment` to reflect this.